events/class_models/event_models.py

```diff
@@ -1,7 +1,67 @@
 from django.db import models
+from django.db.models import Q
+from django.db.models import Count
```

Comment:
you can do it in one line
```suggestion
from django.db.models import Q, Count
```

Reply:
done. thanks 😄